src/schema/objects/columns.yaml

```diff
@@ -42,6 +42,45 @@ age:
     for privacy purposes.
   type: number
   unit: year
+life_cycle_stage:
+  name: life cycle stage
+  display_name: life cycle stage
+  description: |
+    A non-numeric term for a subject's life cycle stage (e.g., 'adult stage'). 
+    Inspired by the
+    [UBERON life cycle stage](http://purl.obolibrary.org/obo/UBERON_0000105).
+  type: string
+  enum:
+    - $ref: objects.enums.adolescent.value
+    - $ref: objects.enums.adult.value
+    - $ref: objects.enums.embryo.value
+    - $ref: objects.enums.infant.value
+    - $ref: objects.enums.juvenile.value
+    - $ref: objects.enums.late_adult.value
+    - $ref: objects.enums.neonate.value
+    - $ref: objects.enums.perinatal.value
+    - $ref: objects.enums.prime_adult.value
+    - $ref: objects.enums.young_adult.value
+age_reference:
+  name: age reference
+  display_name: age reference
+  description: |
+    Non-numeric term of for a time point in a subject's life cycle that serves 
+    as reference for the numeric age information (e.g., 'start of embryo stage').
+  type: string
+  enum:
+    - $ref: objects.enums.start_of_embryo_stage.value
+    - $ref: objects.enums.start_of_gestational_stage.value
+    - $ref: objects.enums.start_of_neonate_stage.value
+    - $ref: objects.enums.end_of_adult_stage.value
+birthdate:
+  name: birthdate
+  display_name: birthdate
+  description: |
+    Date of the start of a subject's neonate stage. Datetime format.
+    Should not be provided or anonymized for human subjects.
+  type: string
+  format: datetime
```

Comment:
Should we require a time here? This has been a bit cumbersome, as it is often the case that you might know the date of birth but not the exact time. This is pretty common in rodent work.

Reply:
agree, if possible it would be nice to allow both: date and datetime
we need to check if date is a supported format in bids

Reply:
@yarikoptic found out: date exists already as format.

Reply:
@robertoostenveld @effigies could we specify:
format: date, datetime